docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -274,7 +291,7 @@ Once the node has bootstrapped, these flags may be removed.
 ### Manually Provisioned Root Snapshot
 
 You can also provision the root snapshot file manually, then start the node without configuring Dynamic Startup.
-A snapshot can be obtained from any Access Node using Flow CLI.
+A snapshot can be obtained from any Access Node using [Flow CLI](https://docs.onflow.org/flow-cli/).
```

Comment:
```suggestion
A snapshot can be obtained from any Access Node using [Flow CLI](/flow-cli/).
```

Reply:
will that work? I didn't see `flow-cli` as a folder under content to make a relative reference. I saw other places in the documentation where flow-cli is referenced using the full url.

Reply:
I think it should be fine. It works for me locally.